modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItTablesApiTest.java

```diff
@@ -406,6 +406,10 @@ public void testCreateDropTable() throws Exception {
             assertNull(ignite.tables().table(TABLE_NAME));
```

Comment:
Let's add full set of tests for sync/async create/alter/drop options. Seems that most of them already exists though.

Reply:
Added another test which checks all API.